packages/scandipwa/src/component/MyAccountDashboard/MyAccountDashboard.style.scss

```diff
@@ -25,8 +25,12 @@
 
     &-AddressesWrapper {
         @include desktop {
-            display: flex;
+            display: inline-flex;
             justify-content: space-between;
+            flex-wrap: wrap;
+            column-gap: 48px;
+            row-gap: 24px;
+            width: 100%;
```

Comment:
Is it necessary to use the `inline-flex` when you put `width: 100%`?
I believe it will appear the same as the block element.

Reply:
Corrected